libraries/protocol/include/steem/protocol/config.hpp

```diff
@@ -234,7 +234,7 @@
 #define STEEM_MAX_TRANSACTION_SIZE            (1024*64)
 #define STEEM_MIN_BLOCK_SIZE_LIMIT            (STEEM_MAX_TRANSACTION_SIZE)
 #define STEEM_MAX_BLOCK_SIZE                  (STEEM_MAX_TRANSACTION_SIZE*STEEM_BLOCK_INTERVAL*2000)
-#define STEEM_SOFT_MAX_BLOCK_SIZE             (2*1024*1024)
+#define STEEM_MAX_BLOCK_SIZE_HF20             (2*1024*1024)
```

Comment:
This name doesn't need to change because we aren't changing the value during the hardfork, just adding a new enforcement case.